README.md

```diff
@@ -1,67 +1,79 @@
 # Alfred API
 
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](
-  https://www.gnu.org/licenses/lgpl-3.0)
-[![Maven Central](https://img.shields.io/maven-central/v/eu.xenit.apix/apix-interface.svg)](
-  http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22eu.xenit.apix%22%20AND%20a%3A%22apix-interface%22)
-[![Jenkins Build Status](https://jenkins-2.xenit.eu/buildStatus/icon?job=Xenit+Github%2Falfred-api%2Fmaster&subject=Jenkins)](
-  https://jenkins-2.xenit.eu/job/Xenit%20Github/job/alfred-api/job/master/)
-
-## Rules for pull requests
+## License
```

Comment:
That's not how you do badges. E.g. see https://github.com/xenit-eu/alfred-telemetry/blob/master/README.md The way it was was correct actually.